Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Services/RNWServiceTest.cs

```diff
@@ -0,0 +1,117 @@
+﻿using FakeItEasy;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning.DTO;
+using UKHO.MaritimeSafetyInformation.Web.Services;
+using UKHO.MaritimeSafetyInformation.Web.Services.Interfaces;
+
+namespace UKHO.MaritimeSafetyInformation.Web.UnitTests.Services
+{
+    [TestFixture]
+    public class RNWServiceTest
+    {
+        private ILogger<RNWService> _fakeLogger;
+        private IRNWRepository _fakeRnwRepository;
+        private RNWService _rnwService;
+        private RadioNavigationalWarnings _fakeRadioNavigationalWarnings;
+        public const string CorrelationId = "7b838400-7d73-4a64-982b-f426bddc1296";
+
+        [SetUp]
+        public void Setup()
+        {
+            _fakeRadioNavigationalWarnings = new(){ WarningType = 1,
+                                                    Reference = "test",
+                                                    DateTimeGroup = DateTime.UtcNow,
+                                                    Summary = "Test1",
+                                                    Content = "test"};
+            _fakeLogger = A.Fake<ILogger<RNWService>>();
+            _fakeRnwRepository = A.Fake<IRNWRepository>();
+
+            _rnwService = new RNWService(_fakeRnwRepository, _fakeLogger);
+        }
+
+        [Test]
+        public async Task WhenRequestToGetWarningTypes_ThenReturnListOfWarningType()
+        {
+            DateTime _fakeDateTime = DateTime.UtcNow;
```

Comment:
What's fake about it?  Also no underscore needed for local variables:
```suggestion
            DateTime dateTime = DateTime.UtcNow;
```

Same for other tests please.

Reply:
done changes

Reply:
Please change this in all tests.  There are 6 more instances of the same issue.